README.md

```diff
@@ -11,22 +11,711 @@
         <img src="https://img.shields.io/badge/code%20style-black-000000.svg" alt="Codestyle Black" height="17">
     </a>
 </p>
+
 # ClashOfClansAPI </br>
 Python Wrapper for SuperCells Clash Of Clans API </br>
 Get Token from https://developer.clashofclans.com/</br>
 
-# Install using pip </br>
+# Install
 
 > pip3 install cocapi
 
 
-# Usage </br>
+# Features and usage examples
 
-```
+### Initialize
+
+Required to set up the class
+
+```python
 from cocapi import CocApi
-token = 'YOUR_API_TOKEN';
+
+token = 'YOUR_API_TOKEN'
 timeout=1 #requests timeout
+
 api=CocApi(token,timeout)
-api.clan_members("#PU8J2RQ")
 ```
+
+
+## Clans
+
+### Information about a Clan
+```python
+api.clan_tag(tag) #example tag "#9UOVJJ9J"
+```
+```text
+{
```

Comment:
@2kofawsome  https://github.com/dear-github/dear-github/issues/166#issuecomment-236342209 can we do a folding here and below for the output? Would make it more readable and beautiful!

Reply:
Folding is very cool, I will definitely use that again